src/irr/asset/CImageWriterPNG.cpp

```diff
@@ -57,22 +57,68 @@ CImageWriterPNG::CImageWriterPNG()
 #endif
 }
 
+template<asset::E_FORMAT outFormat>
+core::smart_refctd_ptr<asset::ICPUImage> getPNGConvertedOutput(const asset::ICPUImage* image)
+{
+	static_assert(!asset::isBlockCompressionFormat<outFormat>(), "Only non BC formats supported!");
```

Comment:
converter should work FROM BC formats (not TO) actually

Reply:
it's correct, any input to any output except BC (it's png)